<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>检验信息</string>
  </property>
  <widget class="QPushButton" name="btn_confirm">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>490</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 13pt &quot;Hei&quot;;</string>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_ID">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 13pt &quot;Hei&quot;;</string>
   </property>
   <property name="text">
    <string>病人ID</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_patientID">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>60</y>
     <width>131</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>30</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>p******</string>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>185</width>
     <height>150</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="QLabel" name="label_ID_3">
      <property name="styleSheet">
       <string notr="true">font: 13pt &quot;Hei&quot;;</string>
      </property>
      <property name="text">
       <string>是否有饮酒史</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QComboBox" name="cb_smoke">
      <item>
       <property name="text">
        <string>是</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>否</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>未知</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_ID_6">
      <property name="styleSheet">
       <string notr="true">font: 13pt &quot;Hei&quot;;</string>
      </property>
      <property name="text">
       <string>是否有输血史</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="label_ID_4">
      <property name="styleSheet">
       <string notr="true">font: 13pt &quot;Hei&quot;;</string>
      </property>
      <property name="text">
       <string>是否有手术史</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QComboBox" name="cb_sur">
      <item>
       <property name="text">
        <string>是</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>否</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>未知</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QComboBox" name="cb_trans">
      <item>
       <property name="text">
        <string>是</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>否</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>未知</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QComboBox" name="cb_drink">
      <item>
       <property name="text">
        <string>是</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>否</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>未知</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_ID_2">
      <property name="styleSheet">
       <string notr="true">font: 13pt &quot;Hei&quot;;</string>
      </property>
      <property name="text">
       <string>是否有吸烟史</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_ID_5">
      <property name="styleSheet">
       <string notr="true">font: 13pt &quot;Hei&quot;;</string>
      </property>
      <property name="text">
       <string>是否有传染病史</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QComboBox" name="cb_infect">
      <item>
       <property name="text">
        <string>是</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>否</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>未知</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLabel" name="label_ID_7">
      <property name="styleSheet">
       <string notr="true">font: 13pt &quot;Hei&quot;;</string>
      </property>
      <property name="text">
       <string>是否有过敏史</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QComboBox" name="cb_allergy">
      <item>
       <property name="text">
        <string>是</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>否</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>未知</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="creat.qrc"/>
 </resources>
 <connections/>
</ui>
